### PR TITLE
Add container app discover

### PR DESCRIPTION
**Pull request by liuqidake**

### CODE REVIEW

File: dist/index.ts [hunk 1]
```diff
@@ -7,8 +7,7 @@ const app: Express = express();
 const port = process.env.PORT || 8080;
 
 const corsOptions = {
-  origin: ['*'],
-  method: ['GET']
+  origin: ['*']
```
Comment (takyyon) [inline dist/index.ts]: why are we updating this?

File: dist/esm/data/frameworks.json [hunk 2]
```diff
@@ -30,6 +30,12 @@
       "outputLocation": "./",
       "configFiles": ["index.html", "Index.html", "default.htm", "default.html", "index.htm"]
     },
+    {
+      "id": "docker-file",
+      "name": "Dockerfile",
+      "outputLocation": "./",
+      "configFiles": ["Dockerfile", "docker-compose.yml", "docker-compose-dev.yml", "docker-compose.images.yml", "docker-stack.yml"]
```
Comment (takyyon) [inline dist/esm/data/frameworks.json]: don't need this update, can we remove this please? 
the only one needed is here - src/data/frameworks.json

File: dist/cjs/data/frameworks.json [hunk 3]
```diff
@@ -30,6 +30,12 @@
       "outputLocation": "./",
       "configFiles": ["index.html", "Index.html", "default.htm", "default.html", "index.htm"]
     },
+    {
+      "id": "docker-file",
+      "name": "Dockerfile",
+      "outputLocation": "./",
+      "configFiles": ["Dockerfile", "docker-compose.yml", "docker-compose-dev.yml", "docker-compose.images.yml", "docker-stack.yml"]
```
Comment (takyyon) [inline dist/cjs/data/frameworks.json]: don't need this update, can we remove this please? 
the only one needed is here - src/data/frameworks.json

File: src/lib/detectors.ts [hunk 4]
```diff
@@ -439,4 +440,10 @@ function isSWAService(frameworks: Record<string, FrameworkMatch>, projectUrl: st
   }
   
   return false;
+}
+
+function isContainerAppService(frameworks: Record<string, FrameworkMatch>, projectUrl: string) {
+  console.log(`Url: ${projectUrl}`);
```
Comment (takyyon) [inline src/lib/detectors.ts]: remove?

File: bundle.js [hunk 5]
```diff
@@ -1,2 +1,2 @@
...
....com/${e}/${n}${t?`#${t}`:''}`,N=require('express'),E=require('cors'),I=N(),J=process.env.PORT||8080;I.use(E({origin:['*']})),I.get('/',(async(e,n)=>{const t=e.query.repo,o=e.query.org;let i=e.query.branch;const a=e.headers['github-token'];if(process.env.GITHUB_TOKEN='',process.env.__GITHUB_PROJECT__='',o&&'string'==typeof o)if(t&&'string'==typeof t){a&&'string'==typeof a?process.env.GITHUB_TOKEN=a:console.warn('Invalid token header'),'string'!=typeof i&&(i='',console.warn('Invalid branch parameter'));try{await P(n,o,t,i)}catch(e){n.status(500),n.json(JSON.parse(JSON.stringify(e)))}}else n.status(400),n.json({message:'Invalid or missing parameters: Repo'});else n.status(400),n.json({message:'Invalid or missing parameters: Org'})})),I.listen(J,(()=>{console.log(`⚡️[server]: Server is running at https://localhost:${J}`)}));const P=(e,n,t,o)=>((e,n,t)=>L(T(e,n,t),[],!0,[]))(n,t,o).then((i=>{((e,n,t,o)=>{const i=T(n,t,o);return Promise.resolve(w(e,i))})(i,n,t,o).then((n=>{e.status(200),e.json({frameworks:i,recommendation:n})})).catch((n=>{console.log('Error on Recommendation: '+n),e.status(500),e.json(n)})).catch((n=>{console.log('Error on framework: '+n),e.status(500),e.json(n)}))}));
```
Comment (takyyon) [inline bundle.js]: remove this change?

Comment (liuqidake) [reply bundle.js]: Those are all somehow autogenerated. I will remove them

File: dist/index.ts (hunk 1 above)
Comment (liuqidake) [reply dist/index.ts]: I think i need to rebase the change. I did not change this file

File: src/lib/detectors.ts (hunk 4 above)
Comment (liuqidake) [reply src/lib/detectors.ts]: I copied that from isSWAService.. I guess we don't need it

File: dist/index.ts (hunk 1 above)
Comment (liuqidake) [reply dist/index.ts]: Done

File: dist/esm/data/frameworks.json (hunk 2 above)
Comment (liuqidake) [reply dist/esm/data/frameworks.json]: Done

File: src/lib/detectors.ts (hunk 4 above)
Comment (liuqidake) [reply src/lib/detectors.ts]: Done

File: bundle.js (hunk 5 above)
Comment (liuqidake) [reply bundle.js]: Done

File: dist/cjs/data/frameworks.json (hunk 3 above)
Comment (liuqidake) [reply dist/cjs/data/frameworks.json]: Done

**Pull request merged**